smart_contracts/contracts/test/auction-bids/src/main.rs

```diff
@@ -57,13 +53,10 @@ fn delegate() {
     let args = runtime_args! {
         ARG_DELEGATOR => delegator,
         ARG_VALIDATOR => validator,
-        ARG_SOURCE_PURSE => account::get_main_purse(),
         ARG_AMOUNT => amount,
     };
 
-    let (purse, _amount): (URef, U512) = runtime::call_contract(auction, METHOD_DELEGATE, args);
-
-    runtime::put_key(DELEGATE_PURSE, purse.into())
+    let (_purse, _amount): (URef, U512) = runtime::call_contract(auction, METHOD_DELEGATE, args);
```

Comment:
Let's also remove these values, as we are just calling for effect.

(and below)